examples/energy-management-app/energy-management-common/include/EnergyEvseDelegateImpl.h

```diff
@@ -33,6 +34,34 @@ using chip::Protocols::InteractionModel::Status;
  */
 CHIP_ERROR GetEpochTS(uint32_t & chipEpoch);
 
+/**
+ * @brief   Helper function to get current timestamp and work out the day of week based on localtime
+ *
+ * @param   reference to hold the day of week as a bitmap as defined by EnergyEvse::TargetDayOfWeekBitmap
+ */
+CHIP_ERROR GetDayOfWeekNow(uint8_t & dayOfWeekMap);
```

Comment:
according to the docs, this should return a `EnergeEvse::TargetOfDayWeekBitmap` instead of a uint8_t

Reply:
This has been changed in the new file EnergyTimeUtils.cpp/h